doc_src/cmds/cd.rst

```diff
@@ -24,14 +24,26 @@ Fish also ships a wrapper function around the builtin ``cd`` that understands ``
 
 As a special case, ``cd .`` is equivalent to ``cd $PWD``, which is useful in cases where a mountpoint has been recycled or a directory has been removed and recreated.
 
+Options
+-------
+
+The following options are available:
+
+- ``-L`` or ``--logical`` Force symoblic links to be followed.
```

Comment:
Typo: `symoblic` should be `symbolic`

Reply:
Shouldn't we simply omit this option, since it's the default? I always find redundant options confusing.

Reply:
It's there for backwards-compatibility if the default is ever switched.

I don't think we will, in this case (given that we did so in 3.0), but I don't think we ever regretted a default option.